recipes-openxt/xenclient/xenclient-config-access/config-access.initscript

```diff
@@ -32,8 +32,7 @@ sig=$(dd if="/dev/mapper/${CONFIG_LV}" bs=4 count=1 2>/dev/null)
 case "$sig" in
 LUKS)
 	keycmd="get-config-key"
-	slot=0
-	eval $keycmd | cryptsetup -q -d - -S $slot luksOpen \
+	eval $keycmd | cryptsetup -q -d - luksOpen \
```

Comment:
If no slot is specified, it will try every defined slot. If you specify a slot, it will only attempt to use key to unlock the specified slot or fail. Taking this approach so that alternate key schemes can place the keys in the slots they prefer and this script will not have to be changed.